src/ripple/app/main/Application.cpp

```diff
@@ -257,6 +258,7 @@ class ApplicationImp
     NodeStoreScheduler m_nodeStoreScheduler;
     std::unique_ptr <SHAMapStore> m_shaMapStore;
     std::unique_ptr <NodeStore::Database> m_nodeStore;
+    PendingSaves pendingSaves_;
```

Comment:
`pendingSaves_` -> `m_pendingSaves` to match the style of the rest of the file.

I think new code should conform to the coding style a file currently has and then if we change the style change it all at once.